### PR TITLE
L2D8A1

### PR DESCRIPTION
**Pull request by evyryan**
Done

### CODE REVIEW

**Comment by evyryan** [comment]
I committed my changes, and this is finished. Not sure how to change the title

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 1]
```diff
@@ -89,12 +89,14 @@
...
                <head xml:id="kzz7yh-f24075-Hd1e151">Quaestio 1</head>
+               <head type="question-title">Utrum demones sunt compositi ex spirituali et corporali substantia</head>
+               <head type="question-title">Change this</head>
                <p xml:id="kzz7yh-f24075-d1e153">
       ¶ Quaestio I. 
       <lb ed="#V" n="7"/>PRimo ostendo quod demones sunt 
       <lb ed="#V" n="8"/>compositi ex 
       spiritua<lb ed="#V" n="9" break="no"/>li et corporali substantia. Aug. 8. de ci. cap. 15. dicit 
-      <lb ed="#V" n="10"/>quod demones sunt aerei. Et. xi. lib. 22. cap. quod 
+      <lb ed="#V" n="10"/>quod demones sunt aerei<!--word?-->. Et. xi. lib. 22. cap. quod
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:99]:
```suggestion
      <lb ed="#V" n="10"/>quod demones sunt aerei. Et. xi. lib. 22. cap. quod 
```

Comment (RobbieBuhite) [reply kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:99]: Yes, "aereus, -a, -um," the adjective meaning "bronze," from "aes, aeris," f., "bronze."

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 2]
```diff
@@ -179,9 +182,9 @@
       <lb ed="#V" n="62"/>spiritus vnitus non vult separari: ergo separatus non vult vniri. 
    </p>
                <p xml:id="kzz7yh-f24075-d1e309">
-                  <lb ed="#V" n="63"/>Contra Auic. x. meta. cap.iet loquens de illo qui est 
-      <lb ed="#V" n="64"/>aptts ad prophetiam: dicit quod hoc est ille in cuius 
-      <lb ed="#V" n="65"/>virsbus animalibus sunt he proprietates scilicet vt audiat verbum 
+                  <lb ed="#V" n="63"/>Contra Auic. x. meta. cap. i. loquens de illo qui est 
+      <lb ed="#V" n="64"/>aptus ad prophetiam: dicit quod hoc est ille in cuius 
+      <lb ed="#V" n="65"/>virtbus<!--word--> animalibus sunt he proprietates scilicet vt audiat verbum
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:187]:
```suggestion
      <lb ed="#V" n="65"/>virtutibus animalibus sunt he proprietates scilicet vt audiat verbum 
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 3]
```diff
@@ -235,12 +238,13 @@
...
                <head xml:id="kzz7yh-f24075-Hd1e413">Quaestio 3</head>
+               <head type="question-title">Utrum demones assumant corpora de elementari natura</head>
                <p xml:id="kzz7yh-f24075-d1e415">
       ¶ Questio III. 
-      <lb ed="#V" n="100"/>Tertio queritur vtrum demons 
+      <lb ed="#V" n="100"/>TErtio queritur vtrum demones 
       <lb ed="#V" n="101"/>assumant 
       corpo<lb ed="#V" n="102" break="no"/>ra de elementari natura. Et videtur quod non Glo. 
-      <lb ed="#V" n="103"/>Gensem 18 dicit de angelis qui apparuerunt 
+      <lb ed="#V" n="103"/>Gensem. <!-- or Genesis?--> 18. dicit de angelis qui apparuerunt
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:247]:
```suggestion
      <lb ed="#V" n="103"/>Genesem. 18. dicit de angelis qui apparuerunt 
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 4]
```diff
@@ -268,7 +272,7 @@
    </p>
                <p xml:id="kzz7yh-f24075-d1e471">
                   <lb ed="#V" n="122"/>Contra Glo. Gen. 28 Abraham angelos videns non 
-      <lb ed="#V" n="123"/>potuisset hospitio suscipere: et cibos 
+      <lb ed="#V" n="123"/>potuisset hospitio suscipere<!-- abbreviation-->: et cibos
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:275]:
```suggestion
      <lb ed="#V" n="123"/>potuisset hospitio suscipere: et cibos 
```

Comment (RobbieBuhite) [reply kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:275]: It's just a colon for punctuation followed by the abbrev. for "et."

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 5]
```diff
@@ -306,9 +310,9 @@
       er<lb ed="#V" n="11" break="no"/>go assumit de natura celesti: restat igitur quod de elementari. 
     </p>
                <p xml:id="kzz7yh-f24075-d1e545">
-                  <lb ed="#V" n="12"/>Ad primum in oppositum dicendum: quo illa Glosu. 
-      <lb ed="#V" n="13"/>vocat celeste elementum aerem. 
-      In<lb ed="#V" n="14" break="no"/>sancta enim scriptura interdum aer dicitur caelum. Unde 
+                  <lb ed="#V" n="12"/>Ad primum in oppositum dicendum: quo illa Glosu.<!-- abbreviation--> 
+      <lb ed="#V" n="13"/>vocat celeste elementum aerem. An
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:314]:
```suggestion
      <lb ed="#V" n="13"/>vocat celeste elementum aerem. In
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 6]
```diff
@@ -346,12 +350,13 @@
             </div>
             <div xml:id="kzz7yh-f24075-Dd1e618">
                <head xml:id="kzz7yh-f24075-Hd1e620">Quaestio 4</head>
+               <head type="question-title">Utrum corpora quae demones assumunt sint perfectibilia ab animal intellectiva<!--endings--></head>
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:353]:
```suggestion
               <head type="question-title">Utrum corpora quae demones assumunt sint perfectibilia a animal intellectiva<!--endings--></head>
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 7]
```diff
@@ -346,12 +350,13 @@
...
                <p xml:id="kzz7yh-f24075-d1e622">
       ¶ Quaestio IIII. 
-      <lb ed="#V" n="42"/>QUarto queritur. vtrum 
-      copo<lb ed="#V" n="43" break="no"/>ra quae 
+      <lb ed="#V" n="42"/>QUarto queritur vtrum 
+      corpo<lb ed="#V" n="43" break="no"/>ra quae 
       demo<lb ed="#V" n="44" break="no"/>nes assumunt sint perfectibilia ab anima 
-      intel<lb ed="#V" n="45" break="no"/>llectiua. Et videtur quod sic. Quia vt habitum est 
+      intel<lb ed="#V" n="45" break="no"/>llectiua<!--ending?-->. Et videtur quod sic. Quia vt habitum est
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:359]:
```suggestion
      intel<lb ed="#V" n="45" break="no"/>llectiua. Et videtur quod sic. Quia vt habitum est 
```

Comment (RobbieBuhite) [reply kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:359]: It's 1st dec. ablative, singular, feminine matching with "anima" after the preposition "ab."

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 8]
```diff
@@ -537,7 +545,7 @@
       <lb ed="#V" n="33"/>cum matre illius pueri concubuisse. 
    </p>
                <p xml:id="kzz7yh-f24075-d1e974">
-                  <lb ed="#V" n="34"/>Contra Glo. Genm. 6 super illud videntes fili dei filias 
+                  <lb ed="#V" n="34"/>Contra Glo. Genm.<!--abbreviation--> 6. super illud videntes filii dei filias
```
Comment (RobbieBuhite) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:548]:
```suggestion
                  <lb ed="#V" n="34"/>Contra Glo. Genesis 6. super illud videntes filii dei filias 
```

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 9]
```diff
@@ -306,9 +309,9 @@
       er<lb ed="#V" n="11" break="no"/>go assumit de natura celesti: restat igitur quod de elementari. 
     </p>
                <p xml:id="kzz7yh-f24075-d1e545">
-                  <lb ed="#V" n="12"/>Ad primum in oppositum dicendum: quo illa Glosu. 
-      <lb ed="#V" n="13"/>vocat celeste elementum aerem. 
-      In<lb ed="#V" n="14" break="no"/>sancta enim scriptura interdum aer dicitur caelum. Unde 
+                  <lb ed="#V" n="12"/>Ad primum in oppositum dicendum: quo illa Glosu.<!-- abbreviation-->
```
Comment (jeffreycwitt) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:312]:
```suggestion
                  <lb ed="#V" n="12"/>Ad primum in oppositum dicendum: quo illa Glossa
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 10]
```diff
@@ -346,10 +349,11 @@
             </div>
             <div xml:id="kzz7yh-f24075-Dd1e618">
                <head xml:id="kzz7yh-f24075-Hd1e620">Quaestio 4</head>
+               <head type="question-title">Utrum corpora quae demones assumunt sint perfectibilia a animal intellectiva<!--endings--></head>
```
Comment (jeffreycwitt) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:352]:
```suggestion
               <head type="question-title">Utrum corpora quae daemones assumunt sint perfectibilia a anima intellectiva</head>
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 11]
```diff
@@ -413,12 +417,14 @@
    </p>
             </div>
             <div xml:id="kzz7yh-f24075-Dd1e750">
+            <!--Left off here-->
```
Comment (jeffreycwitt) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:420]:
```suggestion
```

File: kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml [hunk 12]
```diff
@@ -413,12 +417,14 @@
...
                <head xml:id="kzz7yh-f24075-Hd1e752">Quaestio 5</head>
+               <head type="question-title">Utrum in illis corporibus assumptis possint demones exercere actus virtutis nutritive <!--word?--></head>
```
Comment (jeffreycwitt) [inline kzz7yh-f24075/cod-ve09v2_kzz7yh-f24075.xml:422]:
```suggestion
               <head type="question-title">Utrum in illis corporibus assumptis possint daemones exercere actus virtutis nutritive</head>
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**